scripts/screeps_tencent_batch_rl_runner.py

```
@@ -0,0 +1,783 @@
+  experiment_card.json card-validation.json training-summary.json training-stderr.log report-extract.json \
+  -C repo runtime-artifacts/rl-training runtime-artifacts/rl-simulator
+""".strip()
+        self.ssh_cmd("pack_remote_artifacts", "REMOTE_DIR=" + shlex.quote(self.remote_dir) + " " + bash_lc(remote), timeout=600)
+        local_tar = self.artifact_dir / "remote-artifacts.tar.gz"
+        self.scp_from_worker("download_remote_artifacts", remote_tar, local_tar, timeout=self.args.transfer_timeout_seconds)
+        extract_dir = self.artifact_dir / "remote"
+        if extract_dir.exists():
+            subprocess.run(["rm", "-rf", str(extract_dir)], check=True)
+        extract_dir.mkdir(parents=True, exist_ok=True)
+        with tarfile.open(local_tar, "r:gz") as tar:
+            tar.extractall(extract_dir)
```

Ruff (0.15.12)

[error] Uses of `tarfile.extractall()`

(S202)